x-pack/plugins/encrypted_saved_objects/README.md

```diff
@@ -100,10 +100,10 @@ $ node scripts/jest.js
 
 In one shell, from `kibana-root-folder/x-pack`:
 ```bash
-$ node scripts/functional_tests_server.js --config test/plugin_api_integration/config.js
+$ node scripts/functional_tests_server.js --config test/plugin_api_integration/config.ts
```

Comment:
__nit:__ grr, my fault, forgot to update Readme when moved tests, would you mind fixing this as well while you're here? Thanks!
```suggestion
$ node scripts/functional_tests_server.js --config test/encrypted_saved_objects_api_integration/config.ts
```

Reply:
Done :)